From def0a48bcbd3d2dc8bce7ab30a9c2f20f3a8929f Mon Sep 17 00:00:00 2001
From: Nguyen Tran <nguyen.tran.pz@bp.renesas.com>
Date: Mon, 30 Jun 2025 15:34:50 +0700
Subject: [PATCH] Fix build for Linux 5.10.235

ksys_close has been removed, so use close_fd instead of it.

Upstream-Status: Backport
Signed-off-by: Nguyen Tran <nguyen.tran.pz@bp.renesas.com>
---
 ioctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ioctl.c b/ioctl.c
index e3eefe1..ea6245a 100644
--- a/ioctl.c
+++ b/ioctl.c
@@ -933,7 +933,7 @@ cryptodev_ioctl(struct file *filp, unsigned int cmd, unsigned long arg_)
 		if (unlikely(ret)) {
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(4, 17, 0))
 			sys_close(fd);
-#elif (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
+#elif (LINUX_VERSION_CODE < KERNEL_VERSION(5, 10, 0))
 			ksys_close(fd);
 #else
 			close_fd(fd);
--
2.34.1

